### DISCUSSION TITLE
Supporting cucumber version less than 4

### DISCUSSION THREAD

**Discussion by PavelMal** [allure-framework/allure2#2093] [Questions & Support] [answered]
Hi there!
According to https://repo1.maven.org/maven2/io/qameta/allure/
we support:
 - https://repo1.maven.org/maven2/io/qameta/allure/allure-cucumber3-jvm/
 - https://repo1.maven.org/maven2/io/qameta/allure/allure-cucumber2-jvm/
 - https://repo1.maven.org/maven2/io/qameta/allure/allure-cucumber-jvm/

but in https://github.com/allure-framework/allure-java I can see versions start from 4. 
Do we need to update the information? (add allure cucumber3 and less into https://github.com/allure-framework/allure-java or just delete folders from https://repo1.maven.org/maven2/io/qameta/allure/ )
<img width="952" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/76dd66a1-b2c7-4d6d-b210-3d040e7c6a0c">

**Comment by baev** [comment, marked as answer]
Since release 2.24, we decided to drop support for CucumberJVM versions before version 4 due to low usage stats. Users who require support for versions below 4 may continue using Allure integrations of version 2.23.
